Code/Tools/SceneAPI/SceneCore/Import/SceneImportSettings.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <SceneAPI/SceneCore/Import/SceneImportSettings.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+
+namespace AZ
+{
+    namespace SceneAPI
+    {
+        void SceneImportSettings::Reflect(ReflectContext* context)
+        {
+            if (SerializeContext* serializeContext = azrtti_cast<SerializeContext*>(context); serializeContext)
+            {
+                serializeContext->Class<SceneImportSettings>()
+                    ->Version(0)
+                    ->Field("OptimizeScene", &SceneImportSettings::m_optimizeScene)
+                    ->Field("OptimizeMeshes", &SceneImportSettings::m_optimizeMeshes);
+
+                if (AZ::EditContext* editContext = serializeContext->GetEditContext(); editContext)
+                {
+                    editContext->Class<SceneImportSettings>("Import Settings", "")
+                        ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                        ->Attribute("AutoExpand", true)
+                        ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
+                        ->Attribute(AZ::Edit::Attributes::NameLabelOverride, "")
```

Comment:
Is the `NameLabelOverride` attribute needed?
The actual "Import Settings" itself only shows the children, so the Label override seems unnecessary

Reply:
Yep, definitely seems unnecessary, I'll remove.